src/main/java/se/sundsvall/smloader/integration/db/model/CaseEntity.java

```diff
@@ -5,25 +5,40 @@
 import jakarta.persistence.EnumType;
 import jakarta.persistence.Enumerated;
 import jakarta.persistence.Id;
-import jakarta.persistence.Lob;
 import jakarta.persistence.Table;
+import jakarta.persistence.UniqueConstraint;
 import org.hibernate.Length;
+import org.hibernate.annotations.UuidGenerator;
 
-import java.sql.Clob;
 import java.util.Objects;
 
 @Entity
-@Table(name = "'case'")
+@Table(name = "'case'",
+	uniqueConstraints = {
+		@UniqueConstraint(name = "uq_external_case_id_instance", columnNames = {"external_case_id", "instance"})
+	})
 public class CaseEntity {
 
 	@Id
+	@UuidGenerator
+	@Column(name = "id")
 	private String id;
 
-	@Lob
-	@Column(length = Length.LONG32)
-	private Clob openECase;
+	@Column(name = "family_id")
+	private String familyId;
 
-	@Column(columnDefinition = "varchar(255)")
+	@Column(name = "external_case_id")
+	private String externalCaseId;
+
+	@Column(name = "instance")
+	@Enumerated(EnumType.STRING)
+	private Instance instance;
+
+
+	@Column(name = "open-e-case", length = Length.LONG32)
+	private String openECase;
+
+	@Column(name = "delivery_status")
 	@Enumerated(EnumType.STRING)
 	private DeliveryStatus deliveryStatus;
```

Comment:
I see that the build is failing due to different sort-order on the enum values.

Enums in database are bad in many ways. But enums in Java-code can be nice.

A good approach can be to use `AttributeConverter`. Then you will store your enums as Strings in the database, but but still have them as enums in the entity-classes. (see seab-loader for an example)